src/app/main.cpp

```diff
@@ -104,13 +108,20 @@ int main(int argc, char *argv[])
     // We must save it here because QApplication constructor may change it
     bool isOneArg = (argc == 2);
 
-#if (QT_VERSION < QT_VERSION_CHECK(6, 0, 0)) && !defined(DISABLE_GUI)
+#if !defined(DISABLE_GUI)
+#if (QT_VERSION < QT_VERSION_CHECK(6, 0, 0))
     // Attribute Qt::AA_EnableHighDpiScaling must be set before QCoreApplication is created
     if (qgetenv("QT_ENABLE_HIGHDPI_SCALING").isEmpty() && qgetenv("QT_AUTO_SCREEN_SCALE_FACTOR").isEmpty())
         Application::setAttribute(Qt::AA_EnableHighDpiScaling, true);
     // HighDPI scale factor policy must be set before QGuiApplication is created
     if (qgetenv("QT_SCALE_FACTOR_ROUNDING_POLICY").isEmpty())
         Application::setHighDpiScaleFactorRoundingPolicy(Qt::HighDpiScaleFactorRoundingPolicy::PassThrough);
+#else
+#ifdef Q_OS_WIN
+    if (QOperatingSystemVersion::current() >= QOperatingSystemVersion::Windows10)
+        QApplication::setStyle(u"Fusion"_qs);
+#endif
```

Comment:
> 1. I think you're calling setStyle() too early. From the [docs](https://doc.qt.io/qt-6/qapplication.html#setStyle)

I think you misinterpreted that info. They don't talk that it is incorrect to do it in that way. Moreover...


> 2\. Have you tested that an explicit call to setStyle() can be indeed overriden by the cmd or the ENV var?

Command line parameters (and some kind of environment variables) are processed by QApplication instance so they override the style that is set before instance is created. And Yes, I tested it.

Reply:
> > 1. I think you're calling setStyle() too early. From the [docs](https://doc.qt.io/qt-6/qapplication.html#setStyle)
> 
> I think you misinterpreted that info. They don't talk that it is incorrect to do it in that way. Moreover...
> 

Let me reword my question: Is it intentional to not use the system default color palette?

Reply:
> Is it intentional to not use the system default color palette?

I don't really understand what they meant. Some parts of the Qt documentation are confusing. But it works as expected in the way I've done it. It uses system palette.

Reply:
Apparently there are 2 overloads for setStyle(). My link is about the other overload. The one you're using doesn't have the sentence about color palettes.
It instead has a warning of: "To ensure that the application's style is set correctly, it is best to call this function before the [QApplication](https://doc.qt.io/qt-6/qapplication.html) constructor, if possible."
So you're probably doing the correct thing here.

I am resolving this converstation.

Reply:
> Apparently there are 2 overloads for setStyle(). My link is about the other overload. The one you're using doesn't have the sentence about color palettes.
> It instead has a warning of: "To ensure that the application's style is set correctly, it is best to call this function before the [QApplication](https://doc.qt.io/qt-6/qapplication.html) constructor, if possible."

It's confusing, since the second overload just does `QApplication::setStyle(Qstylefactory::create(styleName))`.